src/comment.rs

```diff
@@ -977,11 +964,16 @@ fn trim_custom_comment_prefix(s: &str) -> String {
 /// Returns `true` if the given string MAY include URLs or alike.
 fn has_url(s: &str) -> bool {
     // This function may return false positive, but should get its job done in most cases.
+    // The regex is indended to capture text such as the below.
+    //
+    // /// An [example].
+    // ///
+    // /// [example]: this::is::a::link
     s.contains("https://")
         || s.contains("http://")
         || s.contains("ftp://")
         || s.contains("file://")
-        || REFERENCE_LINK_URL.is_match(s)
+        || static_regex!(r"^\[.+\]\s?:").is_match(s)
```

Comment:
Continue to refer to the regex as `reference_link_url`. Also, please don't change the content of the original comment. You accidentally added a typo.

```rust
    // A regex matching reference doc links.
    //
    // ```markdown
    // /// An [example].
    // ///
    // /// [example]: this::is::a::link
    // ```
    let reference_link_url = static_regex!(r"^\[.+\]\s?:");

    // This function may return false positive, but should get its job done in most cases.
    s.contains("https://")
        || s.contains("http://")
        || s.contains("ftp://")
        || s.contains("file://")
        || reference_link_url.is_match(s)
```